srcpkgs/docker-engine/template

```diff
@@ -0,0 +1,60 @@
+# Template file for 'docker-engine'
+pkgname=docker-engine
```

Comment:
This package should be named `moby`.

Reply:
docker-engine and moby are synonyms as far as docker is concerned, they have a mirror set up from `docker/engine` to `moby/moby`. I figured I'd use this name so that queries for docker related packages return the correct results. Otherwise something like a dummy package might be required, and calling the package `moby` has no real benefit since nothing else depends on this sort of thing.

Reply:
See my larger response.

Reply:
It looks like the latest tag on `docker/engine` is `v19.03.13`, while the upstream `moby/moby` has many later tags. I don't think switching the upstream makes sense.

Standard Void policy is to use the repo name for the package name, so `moby` is appropriate here. I suppose `docker-cli` is a really special case because `cli` is such a generic noun that it is really devoid of meaning, but `moby` is a proper noun here and identifies the project. Searching for `docker` with `xbps-query` will still turn up `moby` because "Docker" is included in the description.

If there is a compelling reason to use the `docker/engine` upstream or other members disagree about the naming, we can debate the policy.

Reply:
I'll accept this reasoning, updated to `moby`. I'll likely be maintaining this package since I use this in production (yes I'm aware).